experiments/hello-network-socket-ping/local_ping.c

```diff
@@ -0,0 +1,175 @@
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <sys/un.h>
+#include <sys/socket.h>
+
+#include "shared.h"
+#include "util.h"
+
+#define SIZE_MESSAGE_PING 4
+
+const char* g_program_name = "Ping";
+
+// TODO consider rename to: prepare_for_bind_client
+static int
+prepare_for_bind(struct sockaddr_un* addr_ptr)
+{
+    int32_t sock_path_max_len;
+    int32_t ret;
+
+    char error_message[SIZE_ERROR_MESSAGE];
```

Comment:
Please zero `error_message` before use.

Reply:
Done in: https://github.com/arbiter0xf/neotetris/pull/8/commits/c8d23f3e2e9b407564156deb3e69af11b93af847